source/androidx.media3/media3-session/Transforms/Metadata.xml

```diff
@@ -1,31 +1,49 @@
-        />
+<metadata>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaLibraryService.MediaLibrarySession.Builder']/method[@name='setSessionActivity' and count(parameter)=1 and parameter[1][@name='p0']]"        
+        name="managedName"
+		    >
+		    PendingIntent
+	  </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaLibraryService.MediaLibrarySession.Builder']/method[@name='setId' and count(parameter)=1 and parameter[1][@name='p0']]"
+        name="managedName"
+		    >
+		    Id
+	  </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaLibraryService.MediaLibrarySession.Builder']/method[@name='setExtras' and count(parameter)=1 and parameter[1][@name='p0']]"
+        name="managedName"
+		    >
+		    TokenExtras
+	  </attr>
     <remove-node
         path="/api/package[@name='androidx.media3.session']/class[@name='MediaLibraryService.MediaLibrarySession.Builder']/method[@name='build' and count(parameter)=0]"
         />
-    <remove-node
-        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setSessionActivity' and count(parameter)=1 and parameter[1][@type='android.app.PendingIntent']]"
-        />
-    <remove-node
-        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setId' and count(parameter)=1 and parameter[1][@type='java.lang.String']]"
-        />
-    <remove-node
-        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setExtras' and count(parameter)=1 and parameter[1][@type='android.os.Bundle']]"
-        />
-    <remove-node
-        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setBitmapLoader' and count(parameter)=1 and parameter[1][@type='androidx.media3.session.BitmapLoader']]"
-        />
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setSessionActivity' and count(parameter)=1 and parameter[1][@name='p0']]"
+        name="managedName"
+		    >
+		    ActivityPendingIntent
+	  </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setId' and count(parameter)=1 and parameter[1][@name='p0']]"
+        name="managedName"
+        >
+        Id
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setExtras' and count(parameter)=1 and parameter[1][@name='p0']]"
+        name="managedName"
+		    >
+		    Bundle
+	  </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.session']/class[@name='MediaSession.Builder']/method[@name='setBitmapLoader' and count(parameter)=1 and parameter[1][@type='p0']]"
```

Comment:
This is OK, but because we kept other with prefix `Set`

Suggestion just `SetBitmapLoader` or `SetBitmapLoaderWith_______` empty space is type of `parameter[1]`.